docs/guides/file-system-based-automated-bundle-generation.md

```diff
@@ -0,0 +1,116 @@
+# File System Based Automated Bundle Generation
+
+To use the automated bundle generation feature introduced in React on Rails v14.0.0, Please upgrade to use [Shakapacker v6.5.0](https://github.com/shakacode/shakapacker/tree/v6.5.0) at least. If you are currently using webpacker, please follow the migration steps available [here](https://github.com/shakacode/shakapacker/blob/master/docs/v6_upgrade.md).
+
+## Configuration
+
+### Enable nested_entries for Shakapacker
+To use the automated bundle generation feature,  In the `webpacker.yml` file, set 
+
+```yml
+nested_entries: true
+```
+
+For more details, see [Configuration and Code ](https://github.com/shakacode/shakapacker#configuration-and-code) section in [shakapacker](https://github.com/shakacode/shakapacker/).
```

Comment:
Extra space before `]`.

Reply:
Shakapacker should be capitalized (and "react" and "rails" in some places below).